src/organisms/gv-schema-form.js

```diff
@@ -393,7 +395,7 @@ export class GvSchemaForm extends LitElement {
     }
     const keys = this.schema.properties ? Object.keys(this.schema.properties) : [];
     this._ignoreProperties = [];
-    return html`${keys.map((key) => this._renderControl(key))}`;
+    return repeat(keys, (key) => this._renderControl(key));
```

Comment:
Be carefull with this, did you test a form validation in design studio ?

Reply:
Thanks for the tip, I tried in schema forms but not in policy studio, will do it ;)